molo/surveys/tests/test_templatetags.py

```diff
@@ -93,6 +93,13 @@ def setUp(self):
                 slug="linked_survey_title",
                 display_survey_directly=False,
             )
+        self.yourwords_molo_survey_page, yourwords_molo_survey_form_field = \
+            self.create_molo_survey_page(
```

Comment:
Be careful when creating an additional object in the `setUp` method, it may cause side-effects elsewhere in the tests. In this case `test_get_survey_list_default` will fail because it expects a default of 2, not 3 survey types.
You just need to update the test.